tests/integration/telemetry/stackdriver/api/stackdriver_filter_test.go

```diff
@@ -133,6 +133,7 @@ func setupConfig(_ resource.Context, cfg *istio.Config) {
 		return
 	}
 
+	cfg.Values["pilot.traceSampling"] = "100"
```

Comment:
Isn't this test supposed to testing Telemetry API? So shouldn't we add tracing in the Telemetry object

Reply:
i found stackdriver tracing may not work with telemetry api.